demos/src/demo-responsive-flat.js

```diff
@@ -0,0 +1,10 @@
+const oTableEl = document.querySelector('[data-o-table-responsive="flat"]');
```

Comment:
in this file, you've copied the code from the actual source of the module. If it´s in the module's source, you don't need to duplicate it. You should instead import `main.js` and instantiate a new oTable with the template in the demo